dev_tools/tests/test_docs.py

```diff
@@ -7,6 +7,11 @@
 from ..globals.directories import get_xsd_file
 from ..nxdl import iter_definitions
 
+SKIP_FILES_FROM_TESTS = [
```

Comment:
I get why you skipped these files here, but should the doc generation not just work on them if they are valid files?

Reply:
Sorry, my mistake, I just skip the doc generation not all the tests, where some tests validate the nexus data structure.

The files are not presentable to the outer users. I think Doc generation would make this NXDL accessible to the public. As the files are reference app defs for nexus file validator, later I will add further nxdl to make the validator fail. 

And the doc generating does not give validity to the files. NXDL validation performed test in `dev_tools` and `validation` function while writing a nexus file from the NeXus template.

Reply:
"The files are not presentable to the outer users. I think Doc generation would make this NXDL accessible to the public." I don't understand this, but this is probably related to my [comment above](https://github.com/FAIRmat-NFDI/nexus_definitions/pull/280#discussion_r1727040486). The user should never be able to use an NXDL in `dev_tools/tests/data`, independent of the doc generation.